### PR TITLE
Add Fraxswap bnb to dex trades - Ready-to-review

### PR DESCRIPTION
**Pull request by Kndlexi**
Brief comments on the purpose of your changes:

**For Dune Engine V2**

I've checked that:

### General checks:
* [ ] I tested the query on dune.com after compiling the model with dbt compile (compiled queries are written to the target directory)
* [ ] I used "refs" to reference other models in this repo and "sources" to reference raw or decoded tables 
* [ ] if adding a new model, I added a test
* [ ] the filename is unique and ends with .sql
* [ ] each sql file is a select statement and has only one view, table or function defined  
* [ ] column names are `lowercase_snake_cased`
* [ ] if adding a new model, I edited the dbt project YAML file with new directory path for both models and seeds (if applicable)
* [ ] if wanting to expose a model in the UI (Dune data explorer), I added a post-hook in the JINJA config to add metadata (blockchains, sector/project, name and contributor Dune usernames)

### Pricing checks:
* [ ] `coin_id` represents the ID of the coin on coinpaprika.com
* [ ] all the coins are active on coinpaprika.com (please remove inactive ones)

### Join logic:
* [ ] if joining to base table (i.e. ethereum transactions or traces), I looked to make it an inner join if possible

### Incremental logic:
* [ ] I used is_incremental & not is_incremental jinja block filters on both base tables and decoded tables
  * [ ] where block_time >= date_trunc("day", now() - interval '1 week')
* [ ] if joining to base table (i.e. ethereum transactions or traces), I applied join condition where block_time >= date_trunc("day", now() - interval '1 week')
* [ ] if joining to prices view, I applied join condition where minute >= date_trunc("day", now() - interval '1 week')

### CODE REVIEW

File: seeds/dex/trades/dex_trades_seed.csv [hunk 1]
```diff
@@ -119,8 +119,7 @@ avalanche_c,woofi,1,2022-11-08,0xae581cf151eacf9877f628e722514674f3f13b19abcce4e
...
-bnb,biswap,1,2022-11-22,0xaf03946d9b98e5d6e841bb50555208801e48d773a8dfdf1ecc203d943c27c603,350,0x2170ed0880ac9a755fd29b2688956bd959f933f8,6.247796965743473757,0x7130d2a12b9bcbfae4f2634d864a1ee1ce3ead9c,0.43849
\ No newline at end of file
+bnb,fraxswap,1,2022-11-19,0x36f1b32a8115300906f74d3ddf2051d62a226045611a251d84a8a32395ead967,439,0x90C97F71E18723b0Cf0dfa30ee176Ab653E89F40,0.251513613449462899,0xbb4CdB9CBd36B01bD1cBaEBF2De08d9173bc095c,0.001
+bnb,fraxswap,1,2022-10-26,0xe09dd465622225d25f5555214eb8e8e6e74d4a4fa5f6ac51b35f520150011f24,50,0x90C97F71E18723b0Cf0dfa30ee176Ab653E89F40,4.663005228579123357,0xbb4CdB9CBd36B01bD1cBaEBF2De08d9173bc095c,0.018388924302368166
+ethereum,fraxswap,1,2022-07-25,0x36ea8f58b542aed8c3f35ce396da15054e04a522efb25281c8ed0080a081083a,200,0x853d955aCEf822Db058eb8505911ED77F175b99e,6321.896197119067345439,0x3432B6A60D23Ca0dFCa7761B7ab56459D9C964D0,1000
+ethereum,fraxswap,1,2022-08-21,0x39714256ccf47a89e297b726e8de9eec9446949a224d5e2215fd4dc87c0c8969,406,0x3432B6A60D23Ca0dFCa7761B7ab56459D9C964D0,2000.000000000000121322,0x853d955aCEf822Db058eb8505911ED77F175b99e,13014.187897251069443882
```
Comment (Hosuke) [inline seeds/dex/trades/dex_trades_seed.csv:125]: GM @Kndlexi , do you think we can add some seeds for polygon fraxswap as well?

Comment (Kndlexi) [reply seeds/dex/trades/dex_trades_seed.csv]: Added now

File: seeds/dex/trades/dex_trades_seed.csv [hunk 2]
```diff
@@ -119,6 +119,12 @@ avalanche_c,woofi,1,2022-11-08,0xae581cf151eacf9877f628e722514674f3f13b19abcce4e
...
+polygon,fraxswap,1,2022-11-01,0xda324654d84c5f9b67f6947575878eb25f8789153151295d08ee2738847a0750,135,0x45c32fA6DF82ead1e2EF74d17b76547EDdFaFF89,0.008797854552852608,0x0d500B1d8E8eF31E21C99d1Db9A6444d3ADf1270,0.01
+polygon,fraxswap,1,2022-11-01,0x9f4b2967cd1cc64490f2765b6d95c70f8d6df37eaa7f2b441e141b2a41f7a793,188,0x0d500B1d8E8eF31E21C99d1Db9A6444d3ADf1270,1.987969423771805753,0x45c32fA6DF82ead1e2EF74d17b76547EDdFaFF89,1.759300000000000086
```
Comment (Hosuke) [inline seeds/dex/trades/dex_trades_seed.csv:127]: Hi @Kndlexi, we need to use lower-cased address in the spellbook project(as some codes/matchings are case sensitive), I will fix the addresses in the commit.

**Review by Hosuke — APPROVED** [review]
To whom it may concern, I have reviewed this PR and here is the summary:

- Metadata and naming convention looks good ✅
- Config blocks match uniqueness tests in schema files ✅
- Tests and seeds added for the new models ✅
- Incremental filters added when joining large tables ✅
- Contract addresses are lowercased ✅
- Inner join to large tables(*.transactions) as possible ✅

**Review by jeff-dude — APPROVED** [review]

**Comment by jeff-dude** [comment]
great work @Kndlexi & @Hosuke

**Pull request merged**